assets/src/editor/events/ticket-assignments-manager/submit-ticket-assignments-button.js

```diff
@@ -37,11 +40,31 @@ const SubmitTicketAssignmentsButton = ( {
 		beforeEditorClose,
 		closeEditor,
 	} );
+	const onClick = async () => {
+		const { forPersist, id, isNew, modelName } = dateEntity;
+		const path = applyQueryString( modelName );
+		const request = {
+			path: isNew ? path : path + '/' + id,
+			method: isNew ? 'POST' : 'PUT',
+			data: forPersist,
+		};
+
+		try {
+			processChanges();
+			await apiFetch( request );
```

Comment:
this won't add the date to the store though ?

I don't like the idea of mixing persistence and API logic directly in with UI components.
I'm not opposed to having updates being made immediately (as opposed to waiting until the user clicks the WP Post `Update` button) but I think that should be managed by a different layer so to speak.

As well, the "Instant Updates" (changes saved immediately like Google Docs) versus "Deferred Updates" (changes not saved until you click button like current WP admin) paradigm  should be discussed with  support and management first.